config/drupal-vm.settings.php

```diff
@@ -32,3 +32,4 @@
 $settings['jcms_sns_topic_template'] = 'arn:aws:sns:local:000000000000:%s';
 $settings['jcms_sqs_region'] = 'us-east-1';
 $settings['jcms_articles_endpoint'] = 'http://end2end--gateway.elifesciences.org/articles/{id}/versions';
+$settings['jcms_all_articles_endpoint'] = 'https://continuumtest--lax.elifesciences.org/api/v2/articles';
```

Comment:
@giorgiosironi This will need to be a value in your formula if you run this on the end2end tests.